### PR TITLE
Bugfix machine names

### PR DESCRIPTION
**Pull request by telphan**
Connect to #26

### CODE REVIEW

File: layers/meta-resin-jetson/recipes-support/tegra-binaries-prepare/tegra-binaries-prepare_28.1.0.bb [hunk 1]
```diff
@@ -16,7 +16,7 @@ B = "${WORKDIR}/build"
 S = "${WORKDIR}"
 
 DTB_jetson-tx2 = "${SHARED}/kernel/dtb/tegra186-quill-p3310-1000-c03-00-base.dtb"
-DTB_jetson-tx2-skycatch = "${DEPLOY_DIR_IMAGE}/tegra186-tx2-cti-ASG916.dtb"
+DTB_skx2 = "${DEPLOY_DIR_IMAGE}/tegra186-tx2-cti-ASG916.dtb"
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-support/tegra-binaries-prepare/tegra-binaries-prepare_28.1.0.bb]: git commit log should not reference the machineoverrides delewtion as thta is not happening anymore

File: layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch [hunk 2]
```diff
@@ -25,10 +25,10 @@ index 9ecaf38..cf61c1f 100644
  	BOOT_TARGET_DEVICES(BOOTENV_DEV)                                  \
  	\
  	"distro_bootcmd=" BOOTENV_SET_SCSI_NEED_INIT                      \
-+ "setenv resin_kernel_load_addr ${loadaddr};"               \
++ "setenv resin_kernel_load_addr ${loadaddr}; "               \
  		"for target in ${boot_targets}; do "                      \
 -			"run bootcmd_${target}; "                         \
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch]: Is this removal intended? The git log does not mention this change

File: layers/meta-resin-jetson/recipes-kernel/linux/linux-tegra_%.bbappend [hunk 3]
```diff
@@ -8,9 +8,17 @@ RESIN_CONFIGS_append = " compat"
...
 
+RESIN_CONFIGS_skx2_append = " cdc_acm"
+
+RESIN_CONFIGS_DEPS[cdc_acm] = " \
+    CONFIG_TTY=y \
+    "
+RESIN_CONFIGS[cdc_acm] = " \
+    CONFIG_USB_ACM=m \
+    "
+
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-kernel/linux/linux-tegra_%.bbappend]: Can the git log say why we added these?

File: layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch (hunk 2 above)
Comment (telphan) [reply layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch]: no, it is very much not intended.

Comment (telphan) [reply layers/meta-resin-jetson/recipes-bsp/u-boot/u-boot-tegra/0001-Integrate-resin-u-boot.patch]: Ok, the way github shows this diff is a bit confusing, but this line is not actually moved in this patch.

File: layers/meta-resin-jetson/recipes-support/tegra-binaries-prepare/tegra-binaries-prepare_28.1.0.bb (hunk 1 above)
Comment (telphan) [reply layers/meta-resin-jetson/recipes-support/tegra-binaries-prepare/tegra-binaries-prepare_28.1.0.bb]: Fixed

File: layers/meta-resin-jetson/recipes-kernel/linux/linux-tegra_%.bbappend (hunk 3 above)
Comment (telphan) [reply layers/meta-resin-jetson/recipes-kernel/linux/linux-tegra_%.bbappend]: Do you want me to link the issue in the git log? Or do you have something else in mind?

**Comment by telphan** [comment]
@resin-jenkins retest this please

**Comment by telphan** [comment]
@resin-jenkins retest this please
